deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,672 @@
+              },
+              {
+                "name": "csi-volume-group",
+                "repository": "quay.io/ibmcsiblock/csi-volume-group-operator",
+                "tag": "v0.9.1",
+                "imagePullPolicy": "IfNotPresent"
+              },
+              {
+                "name": "livenessprobe",
+                "repository": "k8s.gcr.io/sig-storage/livenessprobe",
+                "tag": "v2.6.0",
+                "imagePullPolicy": "IfNotPresent"
+              }
+            ]
+          }
+        },
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "HostDefinition",
+          "metadata": {
+            "name": "host-definition"
+          },
+          "spec": {
+            "hostDefinition": {
+              "nodeName": "node-name"
+            }
+          }
+        },
+        {
+          "apiVersion": "csi.ibm.com/v1",
+          "kind": "HostDefiner",
+          "metadata": {
+            "name": "host-definer"
+          },
+          "spec": {
+            "hostDefiner": {
+              "allowDelete": true,
+              "dynamicNodeLabeling": false,
+              "repository": "quay.io/ibmcsiblock/ibm-block-csi-host-definer",
+              "tag": "1.11.2",
+              "imagePullPolicy": "IfNotPresent",
+              "affinity": {
+                "nodeAffinity": {
+                  "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                      {
+                        "matchExpressions": [
+                          {
+                            "key": "kubernetes.io/arch",
+                            "operator": "In",
+                            "values": [
+                              "amd64",
+                              "s390x",
+                              "ppc64le"
+                            ]
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        }
+      ]
+spec:
+  displayName: "IBM block storage CSI driver operator"
+  description: |
+    The Container Storage Interface (CSI) Driver for IBM block storage systems enables container orchestrators such as Kubernetes to manage the life cycle of persistent storage.
+
+    This is the official operator to deploy and manage IBM block storage CSI driver.
+
+    For compatibility, prerequisites, release notes, and other user information, see [IBM block storage CSI driver documentation](https://www.ibm.com/docs/en/stg-block-csi-driver).
+
+  keywords:
+    - IBM
+    - BlockStorage
+    - CSI
+  relatedImages:
+    - name: ibm-block-csi-operator
+      image: quay.io/ibmcsiblock/ibm-block-csi-operator:1.11.2
+  version: 1.11.2
+  replaces: ibm-block-csi-operator.v1.10.0
+  maturity: stable
+  maintainers:
+    - name: IBM Block CSI Team
+      email: csi.block1@il.ibm.com
+  minKubeVersion: 1.23.0
```

Comment:
this should be 1.27.0

Reply:
Command to find minKubeVersion related to 1.11.2:

```AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator> gg minKubeVersion | grep 1.11.2
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:204:  minKubeVersion: 1.27.0
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:194:  minKubeVersion: 1.27.0
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator>
```